### PR TITLE
hide empty chart rows as well as empty allele frequency groups when n…

### PR DESCRIPTION
**Pull request by zfisch**
…ecessary

### CODE REVIEW

**Comment by zfisch** [comment]
@falquaddoomi I requested your review on this since you've also worked around this code before

File: website/js/index.js [hunk 1]
```diff
@@ -787,6 +787,7 @@ var VariantDetail = React.createClass({
                     title = "Abbreviated AA Change";
                 }
 
+                // frequency charts are not displayed if they're empty
                 if (rowDescriptor.replace) {
                     rowItem = rowDescriptor.replace(variant, prop);
                     if (rowItem === false) {
```
Comment (falquaddoomi) [inline website/js/index.js]: Minor nitpick, but should the comment from line 790 be for this line instead?

Comment (falquaddoomi) [inline website/js/index.js]: Minor nitpick, but should the comment from line 790 be for this line instead?

**Comment by zfisch** [comment]
@falquaddoomi great suggestions, just updated the PR accordingly. please re-review when you have a chance :)

**Pull request merged**